Classes/Service/MauticSendFormService.php

```diff
@@ -25,6 +26,22 @@ class MauticSendFormService implements SingletonInterface, LoggerAwareInterface
 {
     use LoggerAwareTrait;
 
+    /**
+     * @var array
+     */
+    private $extensionConfiguration;
+
+    /**
+     * @var MauticAuthorizeService
+     */
+    private $authorizeService;
+
+    public function __construct()
+    {
+        $this->extensionConfiguration = GeneralUtility::makeInstance(YamlConfiguration::class)->getConfigurationArray();
+        $this->authorizeService = GeneralUtility::makeInstance(MauticAuthorizeService::class);
```

Comment:
You can use a constructor injection for setting the yamlConfiguration and the authorizeService